### PR TITLE
Added simon game 

### LINKED ISSUE #26: Simon game in js
Plzz assign me this under devincept

### PR DESCRIPTION
**Pull request by siddhi-244**
Fixes #26

### CODE REVIEW

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]
Changes that must be addressed:

- Add a card on the homepage of the website to access this game.
- Add a navigation in your game HTML file to the homepage of the website.
- Stash all the commits into a single commit.
- Rename the CSS and JS file according to your project.

**Comment by siddhi-244** [comment]
> Changes that must be addressed:
> 
> * Add a card on the homepage of the website to access this game.
> * Add a navigation in your game HTML file to the homepage of the website.
> * Stash all the commits into a single commit.
> * Rename the CSS and JS file according to your project.

I have renamed the file according to my game

**Comment by siddhi-244** [comment]
Done please check !

**Comment by siddhi-244** [comment]
@kanishkaa24

File: static/js/game.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+function animatePress(currentColor) {
+  $("#" + currentColor).addClass("pressed");
+  setTimeout(function () {
+    $("#" + currentColor).removeClass("pressed");
+  }, 100);
+}
+
+function playSound(name) {
+  var audio = new Audio("./sounds/" + name + ".mp3");
```
Comment (geekymeeky) [inline static/js/game.js:71]: The path should be `./static/music/Simongame/<filename>.mp3`

File: index.html [hunk 2]
```diff
@@ -86,6 +86,12 @@ <h3 id="pop">Have a break? Play a game!</h3>
                     <img src="static/images/dino/dinogame.png" alt="Dino Game" width="200" height="150">
                 </a>
                 <p class="mt-3"><strong>Dino Game</strong></p>
+
+            </div>
+          <div class="col-12 col-md-3">
+                <a href="simon_game.html"><img width="200" height="150"
+                    src="static/images/Simon_game.png"  alt="image"></a>
```
Comment (geekymeeky) [inline index.html:93]: Kindly change the path of `src` to `"static/images/Simon-Game/Simon_game.png"`

**Review by geekymeeky — APPROVED** [review]
Changes that must be addressed:
- Correct `src` path in the index.html
- Add your sounds to `static/music/Simongame/`
- Correct audio path in `static/js/game.js`

File: static/js/game.js (hunk 1 above)
Comment (siddhi-244) [reply static/js/game.js:71]: But sir I have added sounds in sound folder !

Comment (geekymeeky) [reply static/js/game.js:71]: There's already a folder for music. So, it would be better if you put all your music in the `music` folder.

**Comment by siddhi-244** [comment]
Done sir @geekymeeky

**Review by geekymeeky — APPROVED** [review]

**Review by kanishkaa24 — APPROVED** [review]

**Pull request merged**